libs/product/testing/src/factories/magento/configurable/configurable.factory.ts

```diff
@@ -0,0 +1,24 @@
+import { Injectable } from '@angular/core';
+
+import {
+  DaffModelFactory,
+} from '@daffodil/core/testing';
+import { MagentoProductTypeEnum, MagentoConfigurableProduct } from '@daffodil/product';
+
+import { MockMagentoCoreProduct } from '../core/product.factory';
+
+export class MockMagentoConfigurableProduct extends MockMagentoCoreProduct implements MagentoConfigurableProduct {
+  __typename = MagentoProductTypeEnum.ConfigurableProduct;
+	configurable_options = [];
```

Comment:
Shouldn't this factory up some basic configurable options and variants?

Reply:
It could, yea, I can add that.

Reply:
done